pages/real_time.py

```diff
@@ -0,0 +1,303 @@
+        reset_disabled = False
+        interval_disabled = False
+        start_clicks = 0
+
+        # start processing_job
+        # processing_job_name = star_job()
+        # print(processing_job_name)
+
+        # Start data_generator for Windows
+        subprocess.Popen(process, shell=True)
+
+        # Start data_generator for Amazon linux
+        # subprocess.Popen(process)
+        print('start')
+
+    # Handle pause button click
+    if pause_clicks > 0 and state['running']:
+        state['paused'] = not state['paused']
+        pause_disabled = True if state['paused'] else False
+        start_disabled = False
+        interval_disabled = True
+        pause_clicks = 0
+
+        time.sleep(1)
+        # print(processing_job_name)
+        # stop_job(processing_job_name)
+        # Terminate data_generator
+        file = open('pid.txt', 'r')
+        process_id = file.readline()
+
+        # Start data_generator for Windows
+        subprocess.call(["taskkill", "/PID", process_id, '/F'], shell=True)
+
+        # Start data_generator for Amazon linux
+        # subprocess.run(f"kill {process_id}", shell=True)
+
+        print('stop')
+
+    # Handle reset button click
+    if reset_clicks > 0:
+        state['running'] = False
+        state['paused'] = False
+        start_disabled = False
+        pause_disabled = True
+        reset_disabled = True
+        interval_disabled = True
+        start_clicks = 0
+        pause_clicks = 0
+        reset_clicks = 0
+        start_date, end_date, asset = None, None, None
+        time.sleep(1)
+        children = []
+        #
+        # stop_job(processing_job_name)
+        # Terminate data_generator
+        file = open('pid.txt', 'r')
+        process_id = file.readline()
+
+        # Start data_generator for Windows
+        subprocess.call(["taskkill", "/PID", process_id, '/F'], shell=True)
+
+        # Start data_generator for Amazon linux
+        # subprocess.run(f"kill {process_id}", shell=True)
+
+        # Write empty data
+        with open('data.json', 'w') as f:
+            data = log_example()
+            json_data = json.dumps(data, indent=4)
+            f.write(json_data)
+        print('reset')
+        print('-----------------------------------')
+
+    return (start_disabled, pause_disabled, reset_disabled, interval_disabled, start_clicks, pause_clicks, reset_clicks,
+            asset, children, add_input_clicks)
+
+
+@callback(
+    Output('first-graph-realtime', 'figure'),
+    Output('second-graph-realtime', 'figure'),
+    Output('table-realtime', 'data'),
+    Input('interval-realtime', 'n_intervals'),
+    Input('start-button-realtime', 'n_clicks')
+)
+def update_graph_scatter(n1, start_button_clicks):
+    global table_data
+
+    fig = px.line()
+    try:
+        # Data for components
+        data = json.load(open('data.json'))
+
+        df_for_fig1 = pd.DataFrame(data['charts'][0])
+        df_for_fig2 = pd.DataFrame(data['charts'][1])
+
```

Comment:
Try to rewrite this code assuming that there could be more charts. Use loops, dicts instead of hardcoded variables